### PR TITLE
[Add] EmailService 

### PR DESCRIPTION
**Pull request by samatrhea**
…nfig to config file

### Prerequisites

- [x] I have written a descriptive pull-request title
- [x] I have verified that there are no overlapping [pull-requests](https://github.com/RHEAGROUP/CDP4-WebServices-Community-Edition/pulls) open
- [x] I have verified that I am following the CDP4-SDK [code style guidelines](https://raw.githubusercontent.com/RHEAGROUP/CDP4-WebServices-Community-Edition/master/.github/CONTRIBUTING.md)
- [x] I have provided test coverage for my change (where applicable)

### Description
  - Add email service ready for sending emails to the configured SMPT server
  - Add config settings to the config file
  - Add EmailService to bootstrapper so it is registered for the lifetime of the app

### CODE REVIEW

File: CDP4WebServices.API/config_debug.json [hunk 1]
```diff
@@ -19,10 +19,17 @@
     "IsDbSeedEnabled": true,
     "IsDbRestoreEnabled": false
   },
+  "EmailService": {
+    "Sender":  "CDP4" ,
+    "SMTP": "smpt.cdp4.org",
+    "Port": 587,
+    "UserName": "cdp4postmaseter-username",
```
Comment (lxatrhea) [inline CDP4WebServices.API/config_debug.json:26]: small typo?

File: CDP4WebServices.API/config_release.json [hunk 2]
```diff
@@ -16,9 +16,15 @@
     "DatabaseManage": "cdp4manage",
     "StatementTimeout": 120
   },
+  "EmailService": {
+    "SMTP": "smpt.cdp4.org",
+    "Port": 587,
+    "UserName": "cdp4postmaseter-username",
```
Comment (lxatrhea) [inline CDP4WebServices.API/config_release.json:22]: small typo?

**Review by lxatrhea — APPROVED** [review]

**Review by adrianchivu — APPROVED** [review]

File: CDP4WebServices.API/config_release.json (hunk 2 above)
Comment (samatrhea) [reply CDP4WebServices.API/config_release.json:22]: resolved

**Review by alexatrhea — APPROVED** [review]
Nice and clean

**Pull request merged**